src/engine/test/source/builder/builders/assetBuilderOutput_test.cpp

```diff
@@ -107,28 +104,29 @@ TEST(AssetBuilderOutput, BuildsOperates)
     })"};
 
     auto input = observable<>::create<Event>(
-        [=](auto s)
-        {
-            s.on_next(std::make_shared<json::Document>(R"(
+                     [=](auto s)
+                     {
+                         s.on_next(std::make_shared<json::Document>(R"(
                 {"field":"value"}
             )"));
-            s.on_next(std::make_shared<json::Document>(R"(
+                         s.on_next(std::make_shared<json::Document>(R"(
                 {"field1":"value"}
             )"));
-            s.on_next(std::make_shared<json::Document>(R"(
+                         s.on_next(std::make_shared<json::Document>(R"(
                 {"field":"value"}
             )"));
-            s.on_next(std::make_shared<json::Document>(R"(
+                         s.on_next(std::make_shared<json::Document>(R"(
                 {"field":"value1"}
             )"));
-            s.on_next(std::make_shared<json::Document>(R"(
+                         s.on_next(std::make_shared<json::Document>(R"(
                 {"field":"value"}
             )"));
-            s.on_next(std::make_shared<json::Document>(R"(
+                         s.on_next(std::make_shared<json::Document>(R"(
                 {"field":"value"}
             )"));
```

Comment:
Is this correctly idented?

Reply:
Done with latest format, we may need to fix it